test/Microsoft.Azure.WebJobs.Host.FunctionalTests/Blobs/Listeners/ScanBlobScanLogHybridPollingStrategyTests.cs

```diff
@@ -99,10 +101,12 @@ public void TestBlobListenerWithMultipleContainers()
 
             RunExecuteWithMultiPollingInterval(firstContainerExpectedNames, product, executor, testScanBlobLimitPerPoll / containerCount);
 
+            Thread.Sleep(10);
+
             List<string> secondContainerExpectedNames = new List<string>();
             for (int i = 0; i < 2; i++)
             {
-                firstContainerExpectedNames.Add(CreateAblobAndUploadToContainer(secondContainer));
+                secondContainerExpectedNames.Add(CreateAblobAndUploadToContainer(secondContainer));
```

Comment:
Note this test was wrong before I made my change. It actually failed until I added a Sleep to make sure the second scan saw these blobs as later. I made sure this passed both before and after my change.